### ISSUE TITLE
[Other - Accessibility] Screen reader users may entirely miss legal text. (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#55699]
#### VFS team name
Forms Audit Digitize

#### VFS product name
Form 26-4555

#### VFS feature name
26-4555

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
On Step 6, a screen reader user tabbing through the page is likely to skip over the text in the statement of truth text entirely.

This is an issue Jeana and I have been discussing since the Midpoint Review, and it's a really tricky issue. It's coded accessibly and meets WCAG requirements, and the legal text is discoverable if a screen reader user explores the page content further. But the likely mode of interaction by a screen reader user at this stage of the workflow is just tabbing through interactive elements, which means they'll skip over the important legal text and potentially not be aware of the legal consequences.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
This is something @jeana-adhoc, @joshkimux, and I have been talking through a bunch of possible approaches and I don't think there's a perfect solution. I like [Jeana's idea best](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1621) -- put the legal text inside an additional info component that's open by default rather than closed. For this issue, I'm logging it as a defect-2 that can hopefully be resolved as that component change moves through the design system.

**Update after meeting:**
At the Design System Council team, multiple possible solutions were discussed for exploration. Per @humancompanion-usds, until a better UX solution is found we should adopt a more assertive approach to ensure no users will miss legally-significant text (even if being more assertive can be more annoying for screen reader users).

With that in mind, the solution I would recommend is using `aria-describedby` to associate the statement of truth text with the checkbox:

```
<h3>Statement of truth</h3>

<p id="some-unique-id">
   I certify that I am applying for assistance in acquiring specially adapted housing or special home adaptation grant because of the nature of my service-connected disability. I understand that there are medical and economic features yet to be considered before I am eligible for this benefit, and that I will be notified of the action taken on this application as soon as possible.
</p>
<p>
   I have read and accept the <a aria-label="Privacy policy, will open in new tab" target="_blank" href="/privacy-policy/">privacy policy</a>.
</p>
<va-text-input label="Your full name" class="signature-input hydrated" id="veteran-signature" name="veteran-signature" type="text" required="true" error=""></va-text-input>
<div class="form-checkbox">
   <input id="errorable-checkbox-1" name="veteran-certify" type="checkbox" aria-describedby="some-unique-id">
   <label name="veteran-certify-label" for="errorable-checkbox-1">
      I certify the information above is correct and true to the best of my knowledge and belief.<span class="form-required-span">(*Required)</span>
   </label>
</div>
```

Once we land on a UX approach that's better overall, we should be able to drop the `aria-describedby`. But for now this is a good backstop.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.
